internal/datastore/post.go

```diff
@@ -10,13 +11,55 @@ import (
 
 func (d *delphisDB) PutPost(ctx context.Context, post model.Post) (*model.Post, error) {
 	logrus.Debug("PutPost::SQL Create")
-	found := model.Post{}
-	if err := d.sql.Create(&post).First(&found, model.Post{ID: post.ID}).Error; err != nil {
-		logrus.WithError(err).Errorf("Failed to create a post")
+	if err := d.initializeStatements(ctx); err != nil {
```

Comment:
It appears to me that prepare statements is something we can call at startup and not on every usage of the DB, right?

Reply:
discussed offline. statements are on a connection